### PR TITLE
feat: voicekick command

### PR DESCRIPTION
**Pull request by gregdan3**
The original intention was to entirely remove a user from voice, but that isn't possible with discordrb; see https://github.com/shardlab/discordrb/issues/185

This version moves the user to a specified channel if one is set, to the afk channel if one is set, or throws a failure message otherwise.

### CODE REVIEW

**Review by anna328p — CHANGES_REQUESTED** [review]
Looks great overall! However, probably need to add a permission check of some kind

**Comment by anna328p** [comment]
really appreciate the en_kawaii translations lol

File: modules/util.rb [hunk 1]
```diff
@@ -71,4 +71,22 @@ def self.full_avatar(user)
       event.respond event.channel.slowmode_rate
     end
   end
+
+  command :voicekick, {
+    help_available: true,
+    usage: '.voicekick [user] [channel]',
+    min_args: 1,
+    max_args: 2,
+    required_permissions: %i[manage_users]
```
Comment (anna328p) [inline modules/util.rb:80]: unfortunately, discordrb [only supports setting one, global string](https://github.com/shardlab/discordrb/blob/43da7657731d2da7ff68c332f6967b9a0f158476/lib/discordrb/commands/command_bot.rb#L243) for this feature, which means localization does not work.

so permission checks usually happen manually ([example](https://github.com/arch-community/qbot/blob/7b217bfc059fa81d06bb6184d30e7b52e77afbf7/modules/colors.rb#L143)).

**Review by anna328p — CHANGES_REQUESTED** [review]

**Comment by anna328p** [comment]
the issue is that the message it responds with doesn't get localized because of CommandBot jank

**Comment by gregdan3** [comment]
yup- I see it now
tested and pushed

**Review by anna328p — APPROVED** [review]

**Comment by anna328p** [comment]
the other issue is that qbot ignores commands from other bots. which will be more difficult to resolve

**Comment by gregdan3** [comment]
ooo yikes
Where is that logic implemented? We could throw in a bot ID whitelist for the config

**Comment by anna328p** [comment]
https://github.com/arch-community/qbot/blob/46be0065b333327faeb7700cc3e5394ac0a9ac55/lib/init.rb#L32

**Comment by anna328p** [comment]
https://github.com/shardlab/discordrb/blob/ac58cf5853c5c5b4948e9a26ab305a203fac5651/lib/discordrb/bot.rb#L1279

**Comment by anna328p** [comment]
requesting rebase on main

**Pull request merged**